src/index.md

```diff
@@ -12,6 +12,80 @@ The goal of this guide is to answer questions that developers of the JDK might h
+
+### 3. Find a sponsor
+
+Socializing your change on the mailing lists also prevents the surprise that would otherwise make the community choke on their morning coffee when they see the huge patch in a new, unknown PR. As a new developer in the community you will need to make a few friends that agrees with your change. There are many good reasons to make friends, but the one relevant here is that you'll need a sponsor to approve your change. This is in addition to the reviewers (but can be the same person). The sponsor takes on a larger responsibility for your change than the reviewers, and may need to defend the change in case of later problems. Making people choke on their morning coffee is not the way to make friends.
+
+### 4. Get acquainted with local process
+
+Even though we strive to unify how things are done within the OpenJDK, different areas and projects in the OpenJDK may have slight variations in how they work. Some of these differences are highlighted throughout this guide, some aren't. If you're new to an area, make sure you understand local differences before you proceed. Ask your sponsor who should be your main point of contact through your first developer experience in the OpenJDK.
+
+# Mailing Lists
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [OpenJDK Mailing Lists Manager](https://mail.openjdk.java.net/mailman/listinfo)
+:::
+
+The mailing lists are the key communications mechanism for all OpenJDK work. All participation in an OpenJDK project starts with joining the relevant mailing list. A subscriber to an OpenJDK mailing list is referred to as a [Participant](https://openjdk.java.net/bylaws#participant) in the [Bylaws](https://openjdk.java.net/bylaws). As a general recommendation we suggest to subscribe to [announce](https://mail.openjdk.java.net/mailman/listinfo/announce), [discuss](https://mail.openjdk.java.net/mailman/listinfo/discuss), and the `-dev` lists covering your explicit area of interest. All OpenJDK mailing lists are found here:
+
+> [`mail.openjdk.java.net`](https://mail.openjdk.java.net/mailman/listinfo)
+
+The OpenJDK community is a friendly place. To keep it that way it's important to keep a professional tone in emails and be aware that the community is global. Many different people with different backgrounds collaborate in these lists. Even though English is the required language for all lists, many Participants speak other languages as their native language. A high tolerance for non-perfect English is expected from anyone joining these lists.
```

Comment:
I'd also suggest that we state that it's preferred to use one's real name on the mailing lists. There are a couple people who post from anonymized mailboxes. That makes things difficult, as sometimes the responses look like spam. Also, I'd suggest that we suggest people list their professional affiliation if appropriate (that is, if they're actually doing work on the JDK on behalf of their employer).

Reply:
Yes, very good suggestion! Added in my latest commit.